README.md

```diff
@@ -937,75 +939,66 @@ ctx.run(query[Shirt]).foreach(println)
 
 ## Slick integration
 
-[Slick](http://slick.lightbend.com) doesn't have a separate integration at the moment. You just have to provide a `MappedColumnType` for each database column that should be represented as an enum on the Scala side.
-
-For example when you want the `Enum[Greeting]` defined in the introduction as a database column, you can use the following code
-
+In order to use your enumeratum Enums in Slick tables as columns, you will
+ need to construct instances of `MappedColumnType` and make them available
+ where you define and query your slick tables. In order to more easily
+ construct these instances, the enumeratum-slick integration provides a trait
+ `enumeratum.SlickEnumSupport`. This trait provides a method `mappedColumnTypeForEnum`
+ (and variants) for constructing a mapped column type for your enum. For example
+ if you want to use `Enum[Greeting]` in your slick table, mix in `SlickEnumSupport`,
+ which requires a slick `profile` to be in scope, and construct a mapper
```

Comment:
Can you add a small note here to make it obvious that this trait is typically mixed into where the `Table`s reside?

If I'm getting this right, it might be something like " which requires a slick `profile` to be in scope (typically where your `Table`s are defined).

I'm just hoping to make it obvious that the usage of this trait is different from that of the other integrations, where you would typically mix the support trait into the `Enum[T]`.

Reply:
> If I'm getting this right, it might be something like " which requires a slick profile to be in scope (typically where your Tables are defined).

Sounds good. I'll add a fuller example in that style (with the tables) as well as one in the style of a separate mappers trait `trait CommonColumnMappers { val profile: Profile; ... }`. I think that should cover 99% of how people organize their slick code.